multigeiger/thp_sensor.cpp

```diff
@@ -0,0 +1,33 @@
+// temperature, humidity, pressure sensor (usually a BME280) related code
+
+#include <Arduino.h>
+
+#include <Adafruit_Sensor.h>
+#include <Adafruit_BME280.h>
+
+#include "log.h"
+#include "thp_sensor.h"
+
+int haveBME280 = 0;
+
+float bme_temperature = 0.0;
+float bme_humidity = 0.0;
+float bme_pressure = 0.0;
```

Comment:
we should remove the "bme(280)" later.

the sensor name does not matter for the temperature, pressure, humidity values.